apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -5,6 +5,7 @@ import {
   PositioningProps,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28559&commitId=73cf7489302cc8b682b681475eed35f3a9d552b7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size.chromium.png|6743| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>